include/PICA/pica_frag_config.hpp

```diff
@@ -188,8 +188,8 @@ namespace PICA {
 
 	// Config used for identifying unique fragment pipeline configurations
 	struct FragmentConfig {
-		OutputConfig outConfig;
-		TextureConfig texConfig;
+		OutputConfig outConfig{};
+		TextureConfig texConfig{};
```

Comment:
Nyom

Reply:
wait ill make it even more live fast die young

Reply:
Based